fabric-containers-v0/src/main/java/net/fabricmc/fabric/impl/client/container/ScreenProviderRegistryImpl.java

```diff
@@ -82,8 +81,8 @@ public static void init() {
 						return;
 					}
 
-					AbstractContainerScreen gui = factory.create(syncId, identifier, packetContext.getPlayer(), packetByteBuf);
-					packetContext.getPlayer().container = gui.getContainer();
+					AbstractContainerScreen<?> gui = factory.create(syncId, identifier, packetContext.getExecutor().player, packetByteBuf);
```

Comment:
Getter for the player?

Does it need to be in the exector?

Reply:
Fyi `getExecutor` return result is context specific (`MinecraftClient`). The `getPlayer` method has been removed in favor of `getExecutor().player` (client) or `getNetworkHandler().player` (server) since it is not applicable to login packet contexts.

Reply:
I should probably rename `getExecutor` to `getEngine`